drivers/wifi/esp_at/esp.h

```diff
@@ -109,8 +109,10 @@ extern "C" {
 
 #if defined(CONFIG_WIFI_ESP_AT_VERSION_1_7)
 #define ESP_CMD_CWMODE(mode) "AT+"_CWMODE"="STRINGIFY(_CONCAT(ESP_MODE_, mode))
+#elif defined(CONFIG_WIFI_ESP_AT_VERSION_2_0)
+#define ESP_CMD_CWMODE(mode) "AT+"_CWMODE"="STRINGIFY(_CONCAT(ESP_MODE_, mode))
 #else
-#define ESP_CMD_CWMODE(mode) "AT+"_CWMODE"="STRINGIFY(_CONCAT(ESP_MODE_, mode))",0"
+#define ESP_CMD_CWMODE(mode) "AT+"_CWMODE"="STRINGIFY(_CONCAT(ESP_MODE_, mode))" ,0"
```

Comment:
Please split this in a dedicated commit with a proper description.
It could be useful for instance if one would like to revert this change while keeping the shield definition.

Reply:
Added a commit for it.